mx.sulong/suite.py

```diff
@@ -215,6 +215,14 @@
       ],
       "sha1" : "5552052e53fcd7fe40c558866c9cd51027c17322",
     },
+    "LLVMTESTSUITE" : {
```

Comment:
Can we change that to `LLVM_TEST_SUITE` to match the naming convention in this file?